File: src/main/java/ru/vaschenko/ParkPoint/dto/BookingDto.java
package ru.vaschenko.ParkPoint.dto;

import java.time.LocalDateTime;
import ru.vaschenko.ParkPoint.enams.StateBooking;

public record BookingDto(
        Long id,
        UserDto client,
        ParkingSpaceDto parkingSpace,
        LocalDateTime startTime, 
        LocalDateTime endTime,
        LocalDateTime dateCreated,
        StateBooking status
) {}

File: src/dto/BookingDto.js
export class BookingDto {
    constructor(id, parkingSpace, startTime, endTime, dateCreated, status) {
      this.id = id;
      this.parkingSpace = parkingSpace; // Объект ParkingSpaceDto
      this.startTime = new Date(startTime);
      this.endTime = new Date(endTime);
      this.dateCreated = new Date(dateCreated);
      this.status = status; // Статус бронирования (StateBooking)
    }
  }
  